web/cypress/support/page_objects/onboardingPageNew.ts

```diff
@@ -87,6 +87,13 @@ class OnboardingPageNexusBusiness extends OnboardingSharedElementsWithIndustryQu
   selectEmployeeOrContractorInNJCheckBox() {
     this.getEmployeeOrContractorInNJCheckbox().check();
   }
+
+  getEmployeesInNJCheckbox() {
```

Comment:
This function should live in onOnboardingPageRemoteWorkerBusiness, AND be accessible be onOnboardingPageNexusBusiness

Reply:
You can reference `businessPersonaHelpers.ts` ito see what each foreign business type entails.

Reply:
updated